modules/andes-core/broker/src/main/java/org/wso2/andes/store/rdbms/RDBMSMessageStoreImpl.java

```diff
@@ -93,10 +96,10 @@
                     " FROM " + CONTENT_TABLE +
                     " WHERE " + MESSAGE_ID + " IN (";
 
-    public RDBMSMessageStoreImpl() {
-        queueMap = new ConcurrentHashMap<>();
-        
-    }
+    /**
+     * The cache which holds the queue mappings(queue name to queue id) in memory
+     */
+    private LoadingCache<String, Integer> queueIDs;
```

Comment:
The variable name 'queueIds' says what it holds. Updated the name to 'queueMappings' and updated the comment to indicate how data is loaded from the database in the absence of a queue name.